Specifications/Language/2_Statements/CallStatements.md

```diff
@@ -1,7 +1,7 @@
 # Call statements
 
 
-Call statements are an important part of any programming language. Operation and function calls, much like [partial applications](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/3_Expressions/PartialApplication.md#partial-application), can be used as an expression anywhere as long as the returned value is of a suitable type. However, they can also be used as statements if they return `Unit`. 
+Call statements are an important part of any programming language. Operation and function calls, much like [partial applications](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/3_Expressions/closures.md#partial-application), can be used as an expression anywhere as long as the returned value is of a suitable type. However, they can also be used as statements if they return `Unit`. 
```

Comment:
I think this needs to be `Closures` because of case-sensitivity, otherwise the links don't work on GitHub?

```suggestion
Call statements are an important part of any programming language. Operation and function calls, much like [partial applications](https://github.com/microsoft/qsharp-language/blob/main/Specifications/Language/3_Expressions/Closures.md#partial-application), can be used as an expression anywhere as long as the returned value is of a suitable type. However, they can also be used as statements if they return `Unit`. 
```